package.json

```diff
@@ -67,7 +67,7 @@
     "analyze": "yarn build && source-map-explorer 'build/static/js/*.js' --gzip",
     "build": "yarn build-packages && yarn build-app",
     "build-app": "tsc && yarn save-build-info && vite build && rm build/config.json",
-    "build-packages": "yarn workspaces foreach --include '@terra-ui-packages/*' --parallel run build",
+    "build-packages": "yarn workspaces foreach --include '@terra-ui-packages/*' --parallel --topological-dev run build",
```

Comment:
Packages must now be built in order based on their interdependencies.
https://yarnpkg.com/cli/workspaces/foreach#details